pkg/apis/componentconfig/types.go

```diff
@@ -150,6 +150,9 @@ type KubeletConfiguration struct {
 	// default /var/run/kubernetes). If tlsCertFile and tlsPrivateKeyFile
 	// are provided, this flag will be ignored.
 	CertDirectory string `json:"certDirectory"`
+	// BootstrapKubeConfig is the path to config file or directory of files that
+	// will be used to get client certificate for kubelet.
+	BootstrapKubeconfig string `json:"bootstrapKubeconfig"`
 	// hostnameOverride is the hostname used to identify the kubelet instead
```

Comment:
This should not be in componentconfig. Put it in Options with the other kubeconfig flag.